Day6_excercise.rb

```diff
@@ -0,0 +1,77 @@
+#1.store names of students
+names = ["Jessie", "Wendy", "Kelly"]
+#2.store students' names and their ages
+students = {Jessie: 29, Wendy: 22, Kelly: 33}
+#3. store students' names, ages and their favorit colors.
+students = {
+  Jessie: {age: 29, color: "white"},
+  Wendy: {age: 22, color: "Blue"},
+  Kelly: {age: 33, color: "green"}
+}
+# #4.store hashes with students' names, ages and their favorit colors in array
+students = [
+  {
+    name: "Jessie"
```

Comment:
Do you need to add comma between fields?

Reply:
don't need comma

Reply:
👍